src/jit/lir.h

```diff
@@ -282,6 +282,10 @@ class LIR final
         Range Remove(GenTree* firstNode, GenTree* lastNode);
         Range Remove(ReadOnlyRange&& range);
 
+        void Delete(GenTree* node, BasicBlock* block, Compiler* compiler);
+        void Delete(GenTree* firstNode, GenTree* lastNode, BasicBlock* block, Compiler* compiler);
+        void Delete(ReadOnlyRange&& range, BasicBlock* block, Compiler* compiler);
+
```

Comment:
IMO, the arguments should be in the order: (Compiler*, BasicBlock*, <node or range>). why? it aligns better, and conceptually kinda goes from "biggest" to "smallest" concept, which I like.